.gitignore

```diff
@@ -1,6 +1,8 @@
 .~lock.*
 results/*
 api_results/*
+api-results*/*
```

Comment:
why do we have api-results and api_results?

Reply:
new runs have `api-results-#` as a format.  keeping others for backwards compatability